### PR TITLE
Create video

### PR DESCRIPTION
**Pull request by tanyatrostyan**

### CODE REVIEW

File: src/jade/pages/product_3d.jade [hunk 1]
```diff
@@ -18,10 +18,10 @@ block content
...
     .aligner
       button.close Закрыть
-      .product-centre
-        .product-round
-          img(src="../img/phone_img.png")
-        .control-3d.down
-        .control-3d.right
-        .control-3d.left
-        .control-3d.up
+    .product-centre
```
Comment (freeneps) [inline src/jade/pages/product_3d.jade]: I think it will be better to call it `.product-3d`.

File: src/jade/pages/product_3d.jade [hunk 2]
```diff
@@ -18,10 +18,10 @@ block content
...
+      .product-round
```
Comment (freeneps) [inline src/jade/pages/product_3d.jade]: I think it is redundant wrapper. Image could be placed in parent div.

File: src/jade/pages/video.jade [hunk 3]
```diff
@@ -16,10 +16,18 @@ block content
         li M2 mini
         li M2 note
   .separator.clear
-  .aligner
+  .video.aligner
     button.close Закрыть
     aside.video
```
Comment (freeneps) [inline src/jade/pages/video.jade]: Why aside?

File: src/jade/pages/video.jade [hunk 4]
```diff
@@ -16,10 +16,18 @@ block content
...
-     p Распаковка
-     p Полный обзор
-     p Игры
-     p Мультимедиа
\ No newline at end of file
+      h2 Видео MX4 PRO
+      p Распаковка
```
Comment (freeneps) [inline src/jade/pages/video.jade]: It's more logically to use list here instead of p.

File: src/scss/helpers/_typography.scss [hunk 5]
```diff
@@ -5,6 +5,18 @@ h1 {
   font-family: 'Roboto', sans-serif;
   margin-bottom: 20px;
 }
+h2 {
+  color: $color-gray;
+  font-size: 30px;
+  font-weight: 100;
+  font-family: 'Roboto', sans-serif;
+}
+h3 {
```
Comment (freeneps) [inline src/scss/helpers/_typography.scss]: Don't forget spaces between selectors.

**Comment by freeneps** [comment]
Don't forget link on your page. I need see it from design perspective.

**Comment by tanyatrostyan** [comment]
http://194.28.86.229/meizu-tanyatrostyan/html/product_3d.html

**Pull request merged**